.pre-commit-config.yaml

```diff
@@ -43,7 +43,6 @@ repos:
     rev: 22.8.0
     hooks:
       - id: black
-        language_version: python3.8
```

Comment:
Hmm, why was this removed?

Reply:
I removed this from other PRs as well, even in the main one..
The reason is, it expects `3.8` as the python version, else `black` throws an error...
I personally use `3.9.x` in my virtual environment..
So, it's just to use `pre-commit run --all` without necessarily having python 3.8.x